CHANGELOG.md

```diff
@@ -403,8 +403,10 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed `parameters_to_ignore` not properly set to DDPWrapper ([#7239](https://github.com/PyTorchLightning/pytorch-lightning/pull/7239))
 
 
-- Fixed parsing of `fast_dev_run=True` with the built-in `ArgumentParser` ([#7240](https://github.com/PyTorchLightning/pytorch-lightning/pull/7240))
+- Fixed `apex` not properly instantiated ([#7274](https://github.com/PyTorchLightning/pytorch-lightning/pull/7274))
+
```

Comment:
is it for all `apex` use cases or specifically for ddp_spawn?

Reply:
Cleaned up !